Automattic/woocommerce-payments
# Update enabled currencies list/table design.

Fixes #2096

#### Requested changes

* Remove border-top from the first child of the enabled currencies list
* The default currency should not be editable (remove the edit icon)
* The default currency should use the is-shady card background color to differentiate it from the other rows in the table.
* Reduce the size of the emoji flags to 24px
* keep the trash can icons on the list, (they should be the same grey color as the payments list) but replace the edit icon with the word manage, separated from the delete button without the vertical pipe.

#### Changes proposed in this Pull Request

* Removed border-top for the first-child in the list.
* Removed edit/manage button from the default currency (first-child).
* Set background color of default currency to `$studio-gray-0`.
* Set the size of the flags to 24px.
* Changed pencil icon to "manage", and removed border from trash icon so there's no dividing line. Adjusted other spacing accordingly to allow for items to line up.
* Added flex wrapping and min widths to currency and rate columns to allow for better mobile experience.

#### Testing instructions

* Navigate to WooCommerce > Settings > Multi-Currency.
* Make sure there is no way to edit/delete the default currency.
* Default currency background should be grey.
* Reduce screen size to mobile size to confirm things still lay out in a logical fashion.

-------------------
